src/Compilers/CSharp/Portable/Symbols/Tuples/TupleFieldSymbol.cs

```diff
@@ -116,11 +144,11 @@ internal override DiagnosticInfo GetUseSiteDiagnostic()
             return _underlyingField.GetUseSiteDiagnostic();
         }
 
-        internal override bool RequiresCompletion => _underlyingField.RequiresCompletion;
+        internal sealed override bool RequiresCompletion => _underlyingField.RequiresCompletion;
```

Comment:
>internal sealed override bool RequiresCompletion => _underlyingField.RequiresCompletion; [](start = 8, length = 88)

Do we still need completion support for instances of this type? In what scenarios?

Reply:
I don't think we need completion support on this symbol, but felt it safer to leave.

---
In reply to: [426814713](https://github.com/dotnet/roslyn/pull/44231#discussion_r426814713) [](ancestors = 426814713)

Reply:
> I don't think we need completion support on this symbol, but felt it safer to leave.

I believe we added completion support as a workaround for the bug that you are fixing. If you think there is a good reason to keep this code, then I would say the fix is incomplete. And by keeping this code we cannot really verify whether this is a case.

---
In reply to: [426915300](https://github.com/dotnet/roslyn/pull/44231#discussion_r426915300) [](ancestors = 426915300,426814713)